tests/tests.js

```diff
@@ -79,9 +79,13 @@ var runTests = function runTests(arrayFrom) {
 
 	test('it includes Object.prototype values when it is polluted', function (t) {
 		/* eslint-disable no-extend-native */
-		Object.prototype[1] = 42;
```

Comment:
In that case, let's set `Object.prototype[4]` or something - the entire point of the test is that when `Object.prototype` specifically is polluted, that its value does show up.